spec/rubocop/cop/rspec/describe_class_spec.rb

```diff
@@ -51,6 +51,33 @@
     RUBY
   end
 
+  context 'with non-class constants' do
+    it 'ignores them' do
+      expect_no_offenses(<<-RUBY)
+        describe 'Some::VERSION' do
+          subject { Object.const_get(self.class.description) }
+        end
+      RUBY
+    end
```

Comment:
Maybe add a similar example with a regular class, e.g. `'Some::Thing'`? And also one without namespace `'Something'`.

Reply:
OK, added. Also another tests for empty strings and strings starting with numbers.